### DISCUSSION TITLE
BUG: solution for InvalidPassword

### DISCUSSION THREAD

**Discussion by eyeoncomputing** [SteamRE/DepotDownloader#328] [Q&A]
just tried out DepotDownloader for the first time, but kept getting InvalidPassword
- which as randomly generated 16 characters with lowercase, uppercase, numbers and symbols

generated new random password 
- 16 characters with lowercase, uppercase, numbers and NO symbols this time

then it worked fine.

So there is an issue with passwords using symbols

Hope this helps someone.

**Comment by psychonic** [comment]
https://github.com/SteamRE/DepotDownloader/discussions/186

There is not an issue specific to DepotDownloader and symbols in command arguments. You need to escape them probably for whatever shell you're using.
